### PR TITLE
jQuery Validation: require_from_group (error handling fix)

### PR DESCRIPTION
**Pull request by ghost**
Fixes how WET handles the errors from the method `require_from_group`

Well display only 1 error (first element of the group).

Related: https://github.com/wet-boew/wet-boew/issues/7067 and https://github.com/wet-boew/wet-boew/issues/7936

### CODE REVIEW

**Comment by kodecount** [comment]
Should we add an example to the demo page?

**Comment by duboisp** [comment]
A working example could be always helpful or/and you can describe it in the plugin documentation.

**Comment by kodecount** [comment]
I've added it to the docs since the demo seems long as it is.

File: src/plugins/formvalid/formvalid.js [hunk 1]
```diff
@@ -138,6 +138,7 @@ var componentName = "wb-frmvld",
 						// In this case we will place them in the associated label element
 						errorPlacement: function( $error, $element ) {
 							var type = $element.attr( "type" ),
+								group = $element.hasClass( "require_from_group" ) || $element.attr( "data-rule-require_from_group" ),
```
Comment (nschonni) [inline src/plugins/formvalid/formvalid.js]: The class method of rules in JV was removed awhile ago

Comment (kodecount) [inline src/plugins/formvalid/formvalid.js]: Well remove it then.

**Comment by kodecount** [comment]
Here is a demo: https://kodecount.ca/demos/formvalid/required_group

**Review by lucas-hay — APPROVED** [review]
Approved at PR review meeting.

**Comment by biljanap** [comment]
@kodecount 
Hi there, I wanted to see your demo for group validation, but the page does not exist. Do you still have it?

**Comment by biljanap** [comment]
Would somebody be able to give me an example for this? We don't want to build anything custom if solution is available in WET.
If an example exists, it should be added to Forma validation example page.

**Comment by kodecount** [comment]
Sorry I use it as a testing server and I think I broke it. http://kodecount.ca/demos/formvalid/required_group.php.

**Comment by kodecount** [comment]
Basically right now this exist in WET but it broken I think this was a way to patch this type of validation.

**Comment by kodecount** [comment]
Oh didn’t notice it is merged so should be useable in WET

**Comment by biljanap** [comment]
@kodecount 
Thanks :o)

**Pull request merged**